ob1k-concurrent/src/main/java/com/outbrain/ob1k/concurrent/combiners/Combiner.java

```diff
@@ -1,14 +1,14 @@
 package com.outbrain.ob1k.concurrent.combiners;
 
-import com.outbrain.ob1k.concurrent.*;
-import com.outbrain.ob1k.concurrent.handlers.FutureSuccessHandler;
-import com.outbrain.ob1k.concurrent.handlers.SuccessHandler;
+import com.google.common.util.concurrent.UncheckedExecutionException;
```

Comment:
kind of BC for the previous API - immediate handles allowed typed exceptions as part of their signature (ExecutionException), while j.u.f.Function does not. So for still providing this feature of unboxing exception, I'm using this runtime exception as alternative.

Other options: remove this feature, create by ourselves Unchecked Execution Exception, or plainly just unwrap every RuntimeException being thrown.